platform/javascript/export/export.cpp

```diff
@@ -213,6 +213,32 @@ class EditorExportPlatformJavaScript : public EditorExportPlatform {
 	Ref<ImageTexture> stop_icon;
 	int menu_options;
 
+	enum ExportMode {
+		EXPORT_MODE_NORMAL = 0,
+		EXPORT_MODE_THREADS = 1,
+		EXPORT_MODE_GDNATIVE = 2,
+	};
+
+	String _get_template_name(ExportMode p_mode, bool p_debug) const {
```

Comment:
I'll get started on updating my build scripts to build all those flavours :)

Reply:
Though for the `3.2` branch it's going to be a bit tricky as the current Mono version goes with Emscripten 1.39.7 (patched to 1.39.9 because 1.39.7 is broken), which doesn't work for GDNative.

We could possibly make the standard builds with Emscripten 2.0.10+ (normal, threads, gdnative) and Mono builds with Emscripten 1.39.9 (normal, threads). Or work our way through upstream Mono patches to up support to Emscripten 2.0.10 in the Mono 6.12 branch.